torch-neuronx/training/tp_dp_gpt_neox_20b_hf_pretrain/adamw_fp32_optim_params.py

```diff
@@ -0,0 +1,149 @@
+# coding=utf-8
```

Comment:
Can we move this file out since it has been shared by alot of models

Reply:
Yes. We can remove it and reuse the one in 6.9B.

Reply:
Removed.